Docs/user_guide/model_quantization.rst

```diff
@@ -2,51 +2,51 @@
    :class: hideitem
 .. _ug-model-quantization:
 
-========================
-AIMET Model Quantization
-========================
-Models are generally trained on floating-point hardware like CPUs and GPUs. However, when these trained models are run
-on quantized hardware that support fixed-precision operations, model parameters are converted from floating-point
-precision to fixed precision. As an example, when running on hardware that supports 8-bit integer operations, the
-floating point parameters in the trained model need to be converted to 8-bit integers. It is observed that for some
-models, running on an 8-bit fixed-precision runtime introduces a loss in accuracy due to noise added from the use
-of fixed precision parameters and fixed precision operations.
-
-AIMET provides multiple techniques and tools which help to create quantized models with a minimal loss in accuracy
-relative to floating-point models.
-
-This section provides information on typical use cases and AIMET's quantization features.
-
-Use Cases
+########################
+AIMET model quantization
```

Comment:
Can we make the first character of each word capital in the titles?

Reply:
Al said to use the Microsoft style guide. Microsoft uses sentence-style capitalization in titles. We can discuss varying from that if it's important.